src/livestreamer/plugins/picarto.py

```diff
@@ -5,14 +5,15 @@
 from livestreamer.stream import RTMPStream
 
 API_CHANNEL_INFO = "https://picarto.tv/process/channel"
-RTMP_URL = "{}/?{}/{}"
+RTMP_URL = "rtmp://{}:1935/play/"
+RTMP_PLAYPATH = "golive+{}?token={}"
 
 _url_re = re.compile(r"""
     https?://(\w+\.)?picarto\.tv/[^&?/]
 """, re.VERBOSE)
 
 _channel_casing_re = re.compile(r"""
-    <script>placeStreamChannel\('(?P<channel>[^']+)',[^,]+,[^,]+,'(?P<visibility>[^']+)'\);</script>
+    <script>placeStreamChannelFlash\('(?P<channel>[^']+)',[^,]+,[^,]+,'(?P<visibility>[^']+)',[^,]+\);</script>
```

Comment:
Please take a look at my comments in PR #1332. The matching of `placeStreamChannelFlash` works if a single person is streaming, or `placeStreamChannel` if they are involved with a multistream. The following works for both as long as it isn't a multistream where the primary streamer is offline
`<script>placeStreamChannel(Flash)?\('(?P<channel>[^']+)',[^,]+,[^,]+,'(?P<visibility>[^']+)'(,[^,]+)?\);</script>`
If the page you go to is involved in a multistream and the primary streamer (the one who's page you went to) is offline, but the secondary or tertiary stream is live, it will incorrectly match the second or third stream instead of (correctly) reporting the stream offline.